dapps/marketplace/src/index.js

```diff
@@ -67,7 +89,7 @@ class AppWrapper extends Component {
     if (!ready) return null
     return (
       <ApolloProvider client={client}>
-        <HashRouter>
+        <HashRouter history={browserHistory}>
```

Comment:
Hrm not sure if this will cause unintended side effects... perhaps we can remove this for now and look at it again if the error still isn't fixed with the other changes

Reply:
I still think it would be immensely valuable to have this so Sentry has the proper URL, even if it has no baring on this specific issue.  What're your concerns?  

It's building a history obj using the same dependency of react-router.  Even if that somehow fails and `browserHistory` is undefined, I don't should be an issue.

Reply:
Happy to merge this in a separate PR after more testing. My only concern is it changes URLs from dapp.com/#/listing/123 to dapp.com/listing/123 which would break a lot of stuff, including deploying to non-root URLs like https://originprotocol.github.io/test-builds/

Reply:
also... does Sentry not support hash based URLs? I did a quick search and it said it did...

Reply:
What do you mean it changes URLs?  This is literally the same dependency that react-router already uses and I'm not making any changes to the router's internal, nor the browser's history.  It's only used to listen for location change events.

Right now, they're treated as fragments, not as a first-class URL as they are with the dapp.  But I don't really want to argue this forever, so I'll revert this...